src/plat/pc99/linker.lds

```diff
@@ -39,31 +46,30 @@ SECTIONS
         *(.mbh)
         *(.phys.text)
         *(.phys.data)
-        . = ALIGN(4K);
-    }
+        . = . + 1;
```

Comment:
explain a bit why this . = . + 1 is needed?

Reply:
Without the assignment the linker seems to put this segment at the end of the ELF file after all of the other ones.  I couldn't get it to not do that so this is the smallest increment that still results in the segment to be first.  The old `. = ALIGN(4K)` does the same but pads the ELF file with zeros until a 4k boundary. I ended up concluding that to figure out why . + 1 was needed I would have to go look at the linker source.

Reply:
Please add this as a comment in the file then, so this gets demystified